extensions/tmux-sessioner/src/utils/terminalUtils.ts

```diff
@@ -25,5 +25,5 @@ export async function checkTerminalSetup(callback: (isTerminalSetup: boolean) =>
 
 export async function openTerminal() {
   const localTerminalAppName = await LocalStorage.getItem<string>("terminalAppName");
-  execSync(`open -a ${localTerminalAppName}`);
+  execSync(`open -b ${localTerminalAppName}`);
```

Comment:
@Fatpandac  i think we should change the name to align with this behavior.

```suggestion
  execSync(`open -b ${localTerminalAppBundleId}`);
```


Also in the form we may need to rename the dropdown ID also

```typescript
<Form.Dropdown id="terminalAppBundleId" isLoading={loading}>
```

Reply:
Yes, but I will fix it tomorrow night. I am going to bed now.

Reply:
Just take your time 👍 no rush in here and rest well